Tests/StructuredQueriesMacrosTests/DatabaseFunctionMacroTests.swift

```diff
@@ -126,8 +126,8 @@ extension SnapshotTests {
         }
 
         struct __macro_local_14jsonCapitalizefMu_: StructuredQueriesSQLiteCore.ScalarDatabaseFunction {
-          public typealias Input = [String]
-          public typealias Output = [String]
+          public typealias Input = [String].JSONRepresentation
+          public typealias Output = [String].JSONRepresentation
```

Comment:
This was allowing the generic `callAsFunction` to be called with invalid input when a representation was involved.